netsuite partial fetch documentation (#1818)
salto-io/salto: packages/netsuite-adapter/specific-cli-options.md

# NetSuite Specific CLI options
## Partial Fetch
Salto CLI allows the user to fetch only specific netsuite configuration elements and/or file cabinet items by appending the below CLI parameters following the ```-C``` CLI option:

| Name                           | Default when undefined  | Description
| -------------------------------| ------------------------| -----------

> | netsuite.fetchTarget.filePaths                          | fetch all cabinet files                     | A list of regexes of cabinet file paths. Any file whose path matches any of the regexes will be fetched

Reviewer: regexes -> regular expressions?

Author: fixed

# NetSuite Specific CLI options
## Fetch Target
Salto CLI allows the user to fetch only specific netsuite configuration elements and/or file cabinet items by appending the ```netsuite.fetchTarget.filePaths``` and ```netsuite.fetchTarget.types.[type_name]``` parameters following the ```-C``` CLI option.
When none of the ```fetchTarget``` parameters is specified, Salto will fetch all configuration elements.

| Name                           |  Description
| -------------------------------| ------------------------| -----------
| netsuite.fetchTarget.filePaths | A list of regular expressions of cabinet file paths. Only files with matching paths will be fetched.

> | netsuite.fetchTarget.types.[type_name]  | A list of script id regular. Only records of type ```type_name``` with matching script ids will be fetched. 

Reviewer 2: ```suggestion
| netsuite.fetchTarget.types.[type_name]  | A list of script id regular expressions. Only records of type ```type_name``` with matching script ids will be fetched. 
```

Author: 10x fixed
